src/sonic-yang-models/yang-models/sonic-smart-switch.yang

```diff
@@ -74,6 +84,58 @@ module sonic-smart-switch {
 			/* end of container DPUS_LIST */
 		}
 		/* end of container DPUS */
+
+		container DPU_PORT {
+			description "DPU_PORTs part of config_db.json";
+
+			list DPU_PORT_LIST {
+				description "Name of the DPU port";
+				key "PORT_NAME";
+
+				leaf PORT_NAME {
+					description "Name of the DPU port";
+					type string {
+						pattern "[a-zA-Z]+[0-9]+";
+					}
+				}
+
+				leaf state {
```

Comment:
@vvolam why do we need here, when this is just like a normal sonic data port

Reply:
@prsunny had offline discussion with prince sunny and this is only to keep the DPU to NPU port admin up/down and does not control the DPU online or offline which is controlled by Chassisd (which does not use this DPU_PORT_LIST table).